### PR TITLE
use OS agnostic signal handling in process_pending_deposits 

### PR DESCRIPTION
**Pull request by stfung77**
use OS agnostic signal handling in process_pending_deposits

 - this change uses singnal.singnal() instead of loop.add_signal_handler() to capture and handle signals

addresses - https://github.com/stellar/django-polaris/issues/613

### CODE REVIEW

**Review by JakeUrban — APPROVED** [review]
Lets wait until we test this to merge but LGTM. I thought the solution would be a bit more complicated. Now I'm wondering why asyncio has the `add_signal_handler()` method at all.

**Comment by reecexlm** [comment]
tested this change and received a stack trace that looks to be related to unimplemented rail integration

```
python anchor/manage.py process_pending_deposits
I did receive a potential new or unrelated issue:
  File "C:\polaris\django-polaris\lib\site-packages\polaris\integrations\rails.py", line 166, in poll_pending_deposits
    raise NotImplementedError()
NotImplementedError
(django-polaris) c:\polaris>
```

**Comment by stfung77** [comment]
just got this working on windows. i'll merge now

**Pull request merged**
